core/src/main/java/nl/nn/adapterframework/soap/WsdlGenerator.java

```diff
@@ -633,7 +622,7 @@ protected void message(XMLStreamWriter w, String root, Collection<QName> parts)
 
 	protected void portType(XMLStreamWriter w) throws XMLStreamException {
 		w.writeStartElement(WSDL_NAMESPACE, "portType");
-		w.writeAttribute("name", "PortType_" + getName()); {
+		w.writeAttribute("name", "PortType_" + getName());{
```

Comment:
Oei en hier ook

Reply:
Ja, en als je kijkt naar het grotere plaatje van deze code zie je dat hele methods zo geschreven zijn en dat de structuur van de nested blocks de structuur van de XML volgt die uitgeschreven wordt.

De blocks zorgen voor diepere nesting die de nesting van de XML volgt.

Daarom heb ik het zo gelaten maar op 2 plaatsen is de plaatsing van de `{` veranderd. Deze spatie is per ongeluk weggehaald.